iocage_lib/ioc_json.py

```diff
@@ -2415,6 +2436,20 @@ def json_check_prop(self, key, value, conf, default=False):
                             silent=self.silent,
                             exception=ioc_exceptions.ValidationFailed
                         )
+                elif key == 'mac_prefix':
+                    # Invalid letters - 0,1,3,4,5,7,8,9,B,C,D,F
+                    # Valid letters - 2,6,A,E
+                    if not self.validate_mac_prefix(value):
+                        iocage_lib.ioc_common.logit(
+                            {
+                                'level': 'EXCEPTION',
+                                'message': 'Please specify a valid mac_prefix'
+                                           'which contains any of '
+                                           'the following letters as a '
+                                           'second character from the left '
+                                           'side: "2,6,A,E".'
```

Comment:
@Mrt134 can you think of a better way to say this?

Reply:
Suggest:

```
Invalid mac_prefix. Must match `?X` where ? can be any valid hex digit
(0-9, A-F) and X is one of 2, 6, A or E.
```

Reply:
@Mrt134 please let me know if you think the above suggestion is good. Thank you.